platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -610,6 +619,7 @@ public void wireExternalComponents(
      * @param platformPublisher         the platform publisher to bind
      */
     public void bind(
+            @NonNull final PlatformComponentBuilder builder,
```

Comment:
I'm assuming this PR represents the first step towards the pattern of passing *only* the `builder` into this method?

Reply:
Now that I have merged develop, there are now 6 things we are binding via the builder.